src/SampleWebApp/Controllers/AuthorsMvcController.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    public class AuthorsMvcController : Controller
+    {
+        private AuthorDataContext _context;
+
+        public AuthorsMvcController(AuthorDataContext context)
+        {
+            _context = context;    
+        }
+
+        // GET: AuthorsMvc
+        public IActionResult Index()
+        {
+            return View(_context.Author.ToList());
```

Comment:
The async is a flag that needs to be passed in. 
The UI for scaffolding has a checkbox. If its checked, all these would be async.